climada/entity/impact_funcs/test/test_base.py

```diff
@@ -35,6 +35,23 @@ def test_calc_mdr_pass(self):
         imp_fun.mdd = np.arange(0, 1, 0.1)
         new_inten = 17.2
         self.assertEqual(imp_fun.calc_mdr(new_inten), 0.029583999999999996)
+        
+    def test_set_step(self):
+        """Check default impact function: step function"""
+        imp_fun = ImpactFunc()
+        imp_fun.set_step_ImpF(5, 0, 10)
+        self.assertTrue(np.array_equal(imp_fun.paa, np.ones(4)))
+        self.assertTrue(np.array_equal(imp_fun.mdd, np.array([0, 0, 1, 1])))
+        self.assertTrue(np.array_equal(imp_fun.intensity, np.array([0, 5, 5, 10])))
```

Comment:
If the generalization proposal is accepted this should be updated.

Reply:
done in ef360bf3